src/background/datastore.js

```diff
@@ -159,11 +163,11 @@ class DataStore {
       throw new Error(`no existing item for ${id}`);
     }
 
-    // TODO: call API
     let updated = {
       ...info,
       timePasswordChanged: Date.now(),
     };
+    await browser.experiments.logins.update(updated);
     this._all[updated.guid] = updated;
```

Comment:
Same comments as `add()` on these two lines (try/catch and maybe not directly updating the internal datastore).